.travis.yml

```diff
@@ -1,26 +1,41 @@
 # NB: don't set `language: haskell` here
+language: c
+
+sudo: required
 
 # The following enables several GHC versions to be tested; often it's enough to
 # test only against the last release in a major GHC version. Feel free to omit
 # lines listings versions you don't need/want testing for.
-env:
- - GHCVER=7.4.2
- - GHCVER=7.6.3
- - GHCVER=7.8.4
- - GHCVER=7.10.3
- - GHCVER=8.0.1 TEST_OLDER=NO
+matrix:
+  include:
+   - env: GHCVER=7.4.2
+     os: linux
+   - env: GHCVER=7.6.3
+     os: linux
+   - env: GHCVER=7.8.4
+     os: linux
+   - env: GHCVER=7.10.3
+     os: linux
+   - env: GHCVER=8.0.1 TEST_OLDER=NO
+     os: linux
+   - env: GHCVER=7.8.4
+     os: osx
+   - env: GHCVER=7.10.3
+     os: osx
+   - env: GHCVER=8.0.1 TEST_OLDER=NO
```

Comment:
NB: Although TEST_OLDER is currently disabled because (1dd14ccb91b7719c15c516cea76761bc8a01531e), you didn't implement TEST_OLDER for the Mac OS X installer so you shouldn't imply here that we could turn it on once timeouts are fixed.